### DISCUSSION TITLE
Is there any way to finetune the Inference model?

### DISCUSSION THREAD

**Discussion by Freedomeri** [PaddlePaddle/PaddleOCR#12985] [Q&A]
Thanks for the open-source contribution~

And I want to know is there any way to finetune the recognition Inference model, not the pre-trained model? Like, transforming the Inference model to pre-trained model?
I find in doc that the pre-trained model needs millions level Chinese data to get the same performance as the Inference model, however I won't have such large volume data. I only want to finetune on the final Inference model.

Hope for your reply! Thank you very much~

**Comment by asif-ca** [comment]
> I find in doc that the pre-trained model needs millions level Chinese data to get the same performance as the Inference model

Where did you read this, @Freedomeri?

**Comment by Freedomeri** [comment]
> > I find in doc that the pre-trained model needs millions level Chinese data to get the same performance as the Inference model
> 
> Where did you read this, @Freedomeri?

https://github.com/PaddlePaddle/PaddleOCR/blob/1b1dc7e44fa4cfbb83c53ee7a844d7f7b467b108/doc/doc_ch/FAQ.md
Hello, in this file, 2.11~ 
Maybe you need translator because I haven't found the same instruction in the English version file...

**Comment by ericyeyeye** [comment]
> Thanks for the open-source contribution~
> 
> And I want to know is there any way to finetune the recognition Inference model, not the pre-trained model? Like, transforming the Inference model to pre-trained model? I find in doc that the pre-trained model needs millions level Chinese data to get the same performance as the Inference model, however I won't have such large volume data. I only want to finetune on the final Inference model.
> 
> Hope for your reply! Thank you very much~

I tried this to finetune Simplified Chinese model and it works well.
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/knowledge_distillation_en.md
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_ch/knowledge_distillation.md

**Comment by Freedomeri** [comment]
> > Thanks for the open-source contribution~
> > And I want to know is there any way to finetune the recognition Inference model, not the pre-trained model? Like, transforming the Inference model to pre-trained model? I find in doc that the pre-trained model needs millions level Chinese data to get the same performance as the Inference model, however I won't have such large volume data. I only want to finetune on the final Inference model.
> > Hope for your reply! Thank you very much~
> 
> I tried this to finetune Simplified Chinese model and it works well. [https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/knowledge_distillation_en.md](url) [https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_ch/knowledge_distillation.md](url)

Thank you, maybe I can try it.
